app/schemas/models/interactives/interactive.schema.js

```diff
@@ -20,7 +29,11 @@ const interactiveSchema = {
       },
       then: {
         'properties': {
-          'interactiveData': interactiveTypeSchema.interactiveDraggableOrderingSchema,
+          'insertCodeData': { type: 'null' },
```

Comment:
I was looking at these wondering if there's a way to not have to specify the nullable properties every time. They're so slightly different in every case.
Love what you went with!

Reply:
Yeah this is what I had to do to work with treema and json schem draft 07. 😄